### ISSUE TITLE
trilinos.org Certificate Issue?

### ISSUE THREAD

**Issue by jmgate** [trilinos/Trilinos#2336]
@trilinos/framework 

## Expectations
I'd expect [trilinos.org](https://trilinos.org) to still be up and running, but it seems something's gone awry.

## Current Behavior
At least since sometime last week, there seems to be a problem with the site's certificate.  Navigating there with Safari gives you

<img width="675" alt="screen shot 2018-03-05 at 10 00 44 am" src="https://user-images.githubusercontent.com/20327215/36988627-8e5fc25c-205c-11e8-9f0e-8f10f12465c7.png">

Clicking on "view the certificate" yields

<img width="486" alt="screen shot 2018-03-05 at 10 04 35 am" src="https://user-images.githubusercontent.com/20327215/36988685-b1bffd52-205c-11e8-9662-a5878d2ef7e4.png">

I've confirmed this to be an issue on RHEL and MacOS, with Safari, Firefox, and Chrome, and at least one other developer is seeing the same behavior.  I tried skirting the issue by heading to [trilinos.github.io](https://trilinos.github.io), but it seems our Doxygen pages are still hosted on [trilinos.org](https://trilinos.org), so it didn't do me much good.

## Motivation and Context
Just trying to get to Thyra Doxygen.  I can build it myself locally, but whatever's going on here should probably be dealt with.  There may be something wrong on my end, but I've never had troubles accessing the site before.

**Comment by jwillenbring** [comment]
@jmgate John P is working on this. It works again from outside Sandia, but not inside yet.

**Comment by jennloe** [comment]
This is not working outside Sandia.  I got this message today.  
![image](https://user-images.githubusercontent.com/28164775/37249965-971739a2-24b7-11e8-82b5-6b62232297d2.png)

I'm using MacOS with Chrome. 

This has been an ongoing problem on-and-off since I started looking at the Trilinos webpage a year or so ago.  I've run into it many, many times, both inside and outside Sandia. It did seem to stop for awhile, but the problem is back again.
